app/controllers/concerns/partner_category_filter.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 class PartnerCategoryFilter
-  def initialize(params)
+  def initialize(site, params)
```

Comment:
Not sure what the common pattern is but if we only need the neighbourhood ids, would we pass those rather than the whole site?

Reply:
doesn't passing in an array of integers sort of expose the internal workings of the class and break the abstraction?